### PR TITLE
Patch ncurses to symlink libtinfo to libncurses

### PR DESCRIPTION
**Pull request by smoors**

### CODE REVIEW

**Comment by smoors** [comment]
@wpoely86 review?

File: easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCCcore-5.3.0.eb [hunk 1]
```diff
@@ -25,6 +28,12 @@ configopts = [
     '--with-shared --enable-overwrite --enable-ext-colors --enable-widec --includedir=%(installdir)s/include/ncursesw/'
 ]
 
+# Symlink libtinfo to libncurses
+postinstallcmds = [
+    "ln -s %(installdir)s/lib/libncurses.so %(installdir)s/lib/libtinfo.so",
+    "ln -s %(installdir)s/lib/libncurses.a %(installdir)s/lib/libtinfo.a"
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCCcore-5.3.0.eb]: I think there's a good reason we only do this when building `ncurses` with `dummy`, but I don't recall the details.

Maybe @ocaisa does?

Comment (ocaisa) [reply easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCCcore-5.3.0.eb]: https://github.com/easybuilders/easybuild-easyconfigs/pull/5067 has the context but I really don't remember the details.

**Comment by akesandgren** [comment]
This is somewhat outdated, but no clear decision was made on making this soft link or not.
@boegel, @ocaisa, @smoors

**Comment by boegel** [comment]
No follow-up, so closing this...

**Pull request closed without merging**